src/extensions/layout-selector/index.php

```diff
@@ -46,6 +50,128 @@ function coblocks_layout_selector_categories() {
 function coblocks_layout_selector_layouts() {
 	$layouts = array();
 
+	for ( $i=0; $i <= 100; $i++ ) {
```

Comment:
Wonder if we should have one for testing, if WP_DEBUG is on, or a test var we can state?

Reply:
This was mistakenly committed. I've removed it.